docker/test/stateless/run.sh

```diff
@@ -93,10 +104,54 @@ if [ "$NUM_TRIES" -gt "1" ]; then
     mkdir -p /var/run/clickhouse-server
 fi
 
+# Run a CH instance to execute sequential tests on it in parallel with all other tests.
+if [[ "$RUN_SEQUENTIAL_TESTS_IN_PARALLEL" -eq 1 ]]; then
+  mkdir -p /var/run/clickhouse-server3 /etc/clickhouse-server3 /var/lib/clickhouse3
+  cp -r -L /etc/clickhouse-server/* /etc/clickhouse-server3/
+
+  sudo chown clickhouse:clickhouse /var/run/clickhouse-server3 /var/lib/clickhouse3 /etc/clickhouse-server3/
```

Comment:
2 things about this:
* It'd be nicer to be explicit and call them `/var/lib/clickhouse-sequential` or something similar, and `clickhouse-server-sequential.log` / `clickhouse-server-parallel.log`.
* In the report, we need to know which logs to check for a test, so it'd be nice to have a column that says `Parallel / Sequential` next to the Test time.

WDYT?

Reply:
Let's leave it as is and rename it later if it becomes a problem for people